### ISSUE TITLE
[QUERY] Upload oci driver for database migration oracle to Azure database for PostgreSql using sdk

### ISSUE THREAD

**Issue by click2cloud-shailesh** [Azure/azure-sdk-for-net#13235]
**Query/Question**

Hello there, I am trying to implement database migration for Oracle to Azure database for PostgreSQL using sdk,  
for this i need to upload oci driver. Can you guys help me to implement what next after the following code. 

                    var oci_input = new UploadOCIDriverTaskInput()
                    {
                        DriverShare = new Microsoft.Azure.Management.DataMigration.Models.FileShare(path, UserName, Password)
                    };
                   
                    var oci_taskprop = new UploadOCIDriverTaskProperties
                    {
                        Input = oci_input
                    };

  
**Environment:**
Windows 10 .NET Framework 4.8
 - IDE and version : Visual Studio 2019

**Comment by isra-fel** [comment]
Hi @rgreenMSFT will you be able to help @click2cloud-shailesh migrating data to Azure? Thanks

**Comment by rgreenMSFT** [comment]
In the current preview release of the Oracle to PostgreSQL migration, configuration of the OCI driver is not supported via PowerShell.

**Comment by click2cloud-shailesh** [comment]
Hi @rgreenMSFT, I forget to mention that, I am using **.net sdk**. So I am looking forward to do code on **c#** only. 
Can you please let me know that, .net sdk support OCI driver Configuration. If supported, then how comes to do so?

**Comment by ptrnecka** [comment]
Yes, SDK supports OCI driver configuration. It takes 2 steps to deploy the driver. The first step is to upload OCI ZIP file from your share to DMS storage. And the second step is to install the uploaded OCI ZIP to your DMS instance. You can also run an optional third step to check installed OCI driver (which is useful for validation). 
Please note that when DMS instance is stopped (either manually or automatically due to no migration activity), the corresponding VM is deallocated. And when DMS instance is started again, it gets a fresh VM allocated (with no OCI driver installed). Simply rerun the second step to install the driver again (there is no need to re-upload the ZIP though).
Attaching a code sample with all 3 steps:
[Program.zip](https://github.com/Azure/azure-sdk-for-net/files/4950888/Program.zip)
